### PR TITLE
Expose onDrag callback for iOS and Android

### PR DESCRIPTION
**Pull request by michi-trecker**
Expose onDrag callback for iOS and Android. As mapbox on iOS [does not update coordinates on dragging](https://github.com/mapbox/mapbox-gl-native-ios/issues/251), this adds the calculations.

### CODE REVIEW

**Comment by ferdicus** [comment]
@michi-trecker, could you please rebase with `master`?
Thanks 🙇

**Comment by michi-trecker** [comment]
> @michi-trecker, could you please rebase with `master`?
> Thanks 🙇

I rebased it. Please have a look.

File: javascript/components/PointAnnotation.js [hunk 1]
```diff
@@ -91,6 +91,11 @@ class PointAnnotation extends React.PureComponent {
      * This callback is fired once this annotation has stopped being dragged.
      */
     onDragEnd: PropTypes.func,
+
+    /**
+    * This callback is fired during this annotation is being dragged.
```
Comment (ferdicus) [inline javascript/components/PointAnnotation.js:96]: maybe replace `during` with `while`

**Comment by michi-trecker** [comment]
@mfazekas I added `console.log()` to the example on ShowPointAnnotations. The output looks on Android/iOS like this:
```
[Fri Jul 31 2020 16:42:41.981]  LOG      onDragStart: pointAnnotation1 [-73.99204889086162, 40.73762285928893]
[Fri Jul 31 2020 16:42:42.245]  LOG      onDrag: pointAnnotation1 [-73.99204999641877, 40.73762346937869]
...
[Fri Jul 31 2020 16:42:43.229]  LOG      onDrag: pointAnnotation1 [-73.99309069350511, 40.73650161609086]
[Fri Jul 31 2020 16:42:43.380]  LOG      onDragEnd: pointAnnotation1 [-73.99310678675441, 40.736489421933015]
```
## GIF:
drag_marker

**Comment by ferdicus** [comment]
Tested on both Android and iOS, works fine ✅ 

Thanks again for this 👍

**Pull request merged**